src/libs/ReportUtils.ts

```diff
@@ -1631,7 +1631,7 @@ function isPolicyExpenseChatAdmin(report: OnyxEntry<Report>, policies: OnyxColle
 /**
  * Checks if the current user is the admin of the policy.
  */
-function isPolicyAdmin(policyID: string, policies: OnyxCollection<Policy>): boolean {
+function isPolicyAdmin(policyID: string | undefined, policies: OnyxCollection<Policy>): boolean {
```

Comment:
Also, same here. Can we do an early return here when `policyID` is undefined?

Reply:
I don't think it's necessary to add more code (3 lines) to this. If policyID is undefined, the policy object should be undefined too and the result will be false.